examples/recipes/pagination/queries/first.json

```
@@ -0,0 +1,12 @@
+{
+    "order": [
+        [
+            "Orders.number",
+            "asc"
+        ]
+    ],
```

```suggestion
    "order": {
        "Orders.number": "asc"
    },
```

I think that object-like order option is more human-friendly 😄